packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -1246,11 +1245,11 @@ export class Viewer {
   private updateInteractionApi(previousFrame?: Frame): void {
     if (previousFrame != null && this.frame != null) {
       const hasChangedFromPerspective =
-        FrameCamera.isPerspectiveFrameCamera(previousFrame.scene.camera) &&
-        FrameCamera.isPerspectiveFrameCamera(this.frame.scene.camera);
```

Comment:
Was the previous check wrong? It was looking for perspective in both cases?

Reply:
That it was, fixed this in #351 but figured it made sense to fix with this change as well